### ISSUE TITLE
Would be nice to have some easy-to-remember simple option to enable all checks :-)

### ISSUE THREAD

**Issue by hughperkins** [jrprice/Oclgrind#110]
As far as I know, to enable all checks I hvaeto remember how to write:
```
oclgridn --check-something --check-something-else --cant-remmber-this-one-either myprogram
```
Can we have some simple option, like:
```
oclgrind --all myprogram
```
?

**Comment by hughperkins** [comment]
:-)

**Comment by jrprice** [comment]
Sounds reasonable. We would probably want to only enable a subset of all the plugins though (you probably wouldn't want this to dump instruction histograms and enable interactive mode with this).

**Comment by hughperkins** [comment]
Yeah.  I'm currently using the options `oclgrind --check-api --data-races --uniform-writes --uninitialized`.  I dont know if those are the apporpriate options I should be using, for general sanity checking that my program isnt doing anything strange?

**Comment by UniqueFool** [comment]
> you probably wouldn't want this to dump instruction histograms and enable interactive mode with this

actually, that sounds like it could be very interesting/useful for the ML use-case I mentioned at: https://github.com/jrprice/Oclgrind/issues/109
